workflow/executor/docker/docker.go

```diff
@@ -86,6 +89,37 @@ func (d *DockerExecutor) GetOutputStream(containerID string, combinedOutput bool
 	return reader, nil
 }
 
+func (d *DockerExecutor) GetExitCode(containerID string) (int32, error) {
+	cmd := exec.Command("docker", "inspect", containerID, "--format='{{.State.ExitCode}}'")
+	log.Info(cmd.Args)
+	reader, err := cmd.StdoutPipe()
+	if err != nil {
+		return -1, errors.InternalWrapError(err)
+	}
+	err = cmd.Start()
+	if err != nil {
+		return -1, errors.InternalWrapError(err)
```

Comment:
Having a string that would identify this error location would make debugging easier. Something such as 
```suggestion
		return 0, errors.InternalWrapError(err, "Could not start command")
```

Reply:
Please do the same for other error returns